File: CursoCsharpUdemy/exer/outros/listaAlgoritimos/algoritimo/Exer02.cs
/*
*   2. Faça um algoritmo para calcular quantas ferraduras
*   são necessárias para equipar todos os 
*   cavalos comprados para um haras.
*/

using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace exer.outros.listaAlgoritimos.algoritimo
{
    public class Exer02
    {
        static void Main(string[] args) 
        {
            double horseshoe, horse, paw = 4, total;

            Console.WriteLine("Informe quantos cavalos foram comprados: ");
            horse = double.Parse(Console.ReadLine());

            total = paw * horse;

            Console.WriteLine("Total de ferraduras necessárias: " + total);
        }
    }
}

File: CursoJava/Exer/Algoritimo/src/Exer02.java
/*
2. Faça um algoritmo para calcular quantas ferraduras são necessárias para equipar todos os
cavalos comprados para um haras.
 */

import java.util.Scanner;

public class Exer02 {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.println("Digite a quantidade de cavalos: ");
        int cavalos = sc.nextInt();

        int ferraduras = cavalos * 4;

        System.out.println("A quantidade de ferraduras necessárias é: " + ferraduras);

        sc.close();
    }
}
